frontend/src/theme/components/Text.ts

```diff
@@ -45,6 +45,10 @@ const Text = {
       margin: "12px",
       width: "100%",
     },
+    link: {
+      color: "blue.500",
+      cursor: "pointer",
```

Comment:
there is a `Link `component from chakraui that I think you'd find useful: https://chakra-ui.com/docs/navigation/link
You can use it instead of this Text variant